src/components/dataset/details/CreateFullSnapshotNamingView.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  TextField,
+  Typography,
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = (theme) => ({
+  root: {
+    padding: theme.spacing(2),
+  },
+  textField: {
+    marginBottom: theme.spacing(1),
+  },
+  buttonContainer: {
+    justifyContent: 'space-between',
+  },
+});
+
+const CreateFullSnapshotNamingView = ({
+  classes,
+  description,
+  onDismiss,
+  name,
+  setDescription,
+  setIsSharing,
+  setName,
+}) => {
```

Comment:
Linter's not happy here. Maybe take out the return so we don't get spammed with this in other PRs